opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -113,6 +132,11 @@
          */
         SpanBuilder addReference(String referenceType, SpanContext referencedContext);
 
+        /**
+         * Do not create an implicit {@link References#CHILD_OF} reference to the {@link ActiveSpanSource#activeSpan}).
+         */
+        SpanBuilder ignoringActiveSpan();
```

Comment:
I was thinking about the `withXYZ` methods... will change, though, since I don't care much.